asapdiscovery/data/utils.py

```diff
@@ -998,5 +1000,25 @@ def load_exp_from_sdf(fn):
     return exp_data_compounds
 
 
+def check_filelist_has_elements(filelist: Union[glob.glob, List], tag: Optional[str]="untagged") -> None:
+    """
+    Check that a glob or list of files actually contains some elements - if not, raise an error.
+
+    Parameters
+    ----------
+    filelist : Union[glob.glob, List]
+        List of files or glob
+    tag : Optional[str]
+        Tag to add to error message if list is empty
+    
+    Returns
+    -------
+    None
+    """
+    if len(filelist) == 0:
+        raise ValueError(f"list of files or glob with tag: {tag} does not contain any elements")
+    
+
+
```

Comment:
this is a good check, glad that this is in here now. I agree this is a good way to implement some first tests - I'll let you know if I think of any other simple ones we can sprinkle throughout.